gallery/src/components/NxTable/NxTableFilterExample.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow,
+  NxFilterInput
+} from '@sonatype/react-shared-components';
+import { toLower } from 'ramda';
+
+const initialState = [
+  {name: 'Anna', country: 'USA'},
+  {name: 'Lean', country: 'France'},
+  {name: 'Louis', country: 'France'},
+  {name: 'Zach', country: 'Colombia'},
+  {name: 'Jimmy', country: 'Germany'}
+];
+
+const NxTableFilterExample = () => {
+
+  const [rows, setRows] = useState(initialState);
```

Comment:
For simplicity, I wouldn't store the rows in the state, I'd just filter the hard-coded data (`initialState`) and store the result in a local variable on every render.

Reply:
I wanted to be consistent with the example for table sorting: https://github.com/sonatype/sonatype-react-shared-components/blob/master/gallery/src/components/NxTable/NxTableSortableExample.tsx#L20